src/plots/polar/constants.js

```diff
@@ -33,6 +33,9 @@ module.exports = {
     cornerLen: 25,
     cornerHalfWidth: 2,
 
-    MINDRAG: 8,
-    MINZOOM: 20
+    // smallest radial distance [px] allowed for a zoombox
+    MINZOOM: 20,
+    // distance [px] off (r=0) or (r=radius) where we transition
+    // from single-sided to two-sided radial zoom
+    OFFEDGE: 20
```

Comment:
per https://github.com/plotly/plotly.js/pull/2200#issuecomment-357393518

> I could imagine decreasing the snap length a little bit

I made this smaller than last Friday's version.

Reply:
> I made this smaller than last Friday's version.

Looks good. I wish there was a way to indicate to the user which behavior they're going to get, like we do with single- or double-ended arrows for the cartesian axis draggers... but I don't see any obvious choices among built-in cursors. I suppose we could do some super-complicated thing where we choose among all eight single-ended arrows depending on which direction you are from the center... but that might even be more confusing since it would make it look like those eight sectors had differences in behavior. Lets leave it as is for now, and see how it feels as it starts getting used.